telegram/ext/updater.py

```diff
@@ -137,14 +139,16 @@ def start_polling(self,
             Queue: The update queue that can be filled from the main thread
 
         """
+        total_timeout = deprecate_network_delay(timeout, total_timeout, **kwargs)
+
         with self.__lock:
             if not self.running:
                 self.running = True
 
                 # Create & start threads
                 self._init_thread(self.dispatcher.start, "dispatcher")
                 self._init_thread(self._start_polling, "updater", poll_interval, timeout,
-                                  network_delay, bootstrap_retries, clean)
+                                  total_timeout, bootstrap_retries, clean)
```

Comment:
This won't be backwards compatible - why would we continue to support network_delay on getUpdates and not here?